<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_settings_advanced_widget</class>
 <widget class="QWidget" name="display_settings_advanced_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display advanced</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Modify these settings only if you have trouble launching the game</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="texcompressionCheckBox">
         <property name="text">
          <string>Force texture compression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="vertexshaderCheckBox">
         <property name="text">
          <string>Disable vertex shaders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="verticesagpCheckBox">
         <property name="text">
          <string>Disable AGP for vertices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pixelshadersCheckBox">
         <property name="text">
          <string>Disable pixel shaders</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>293</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
